dist/index.html

```diff
@@ -8,24 +8,26 @@
 <script defer src="main.js"></script></head>
 <body>
     <main class="container">
-        <section>
-            <h1>Leaderboard</h1><br/>
-            <div class="section-side1">
-                <p id="parag">Recent scores</p>
-                <button class="btn-1">Refresh</button>
-            </div><br/>
-            <div class="section-side2">
+        <h1 id="h1">LeaderBoard</h1>
```

Comment:
- [ ] Your README file does look professional to some level but needs to contain all that's needed for running the app with the API(s) it consumes. It should have instructions about how to clone, install, and any other details needed to run the app. This information should be provided knowing that both non-technical and technical users will use it.
- [ ] Please add license section of this README. Ensure when clicked, it take the user to the license page on github. 

You README.md **_Getting started section_**
![image](https://user-images.githubusercontent.com/84156714/217850496-1c14113a-bf41-4001-96b3-e6e61e28fbae.png)

Please edit this section showing how one can use this repo.